frontend/src/app/sessions/session/session.component.html

```diff
@@ -70,7 +73,7 @@
     (cdkDragEnded)="dragStop()"
   >
     <div
-      class="flex gap-2 rounded-t p-2 cursor-grab active:cursor-grabbing items-center justify-between"
+      class="flex gap-2 rounded-t cursor-grab active:cursor-grabbing items-center justify-between"
```

Comment:
```suggestion
      class="flex gap-2 rounded-t p-2 cursor-grab active:cursor-grabbing items-center justify-between"
```

Was this intentional? Without the padding it looks very compressed and less modern.

Reply:
The main objective is to maximize the available workspace in the session viewer, which is why the padding was initially removed. However, I agree that some level of padding is beneficial, so I've adjusted it to p-1.